package.json

```diff
@@ -2,11 +2,11 @@
   "name": "node_js_app",
   "version": "1.0.0",
   "description": "node.js app",
-  "main": "src/index.js",
+  "main": "src/index.ts",
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon --watch src"
+    "start:dev": "npx nodemon"
```

Comment:
So, if you have start:dev you should have build and start scripts for prod.

Reply:
fixed